DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionNavBarPopoverManager.swift

```diff
@@ -191,6 +191,7 @@ final class NetworkProtectionNavBarPopoverManager: NetPPopoverManager {
                                          activeSitePublisher: activeSitePublisher,
                                          forMenuApp: false,
                                          vpnSettings: vpnSettings,
+                                         proxySettings: proxySettings,
```

Comment:
We're passing around proxy settings because that's where we'll store if the proxy can run or not.